services/motion/builtin/builtin_test.go

```
@@ -639,7 +638,9 @@ func TestMoveOnMapSubsequent(t *testing.T) {
 		entry1GoalLine = strings.ReplaceAll(entry1GoalLine, ":", "\":")
 
 		posepb := &commonpb.Pose{}
-		json.Unmarshal([]byte(entry1GoalLine), posepb)
+		err := json.Unmarshal([]byte(entry1GoalLine), posepb)
+		test.That(t, err, test.ShouldBeNil)
```

this was ignoring an error